blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpAnnotationsIntegrationTest.java

```diff
@@ -50,58 +59,92 @@ public static void setUpHttp() {
                 .withMapping("annotations", EsSetup.fromClassPath("events_mapping.json")));
         eventsIO = new EventElasticSearchIO(esSetup.client());
         httpQueryService = new HttpQueryService();
-        HttpMetricDataQueryServer server = new HttpMetricDataQueryServer();
-        server.setEventsIO(eventsIO);
-        httpQueryService.setServer(server);
+        HttpMetricDataQueryServer queryServer = new HttpMetricDataQueryServer();
+        queryServer.setEventsIO(eventsIO);
+        httpQueryService.setServer(queryServer);
         httpQueryService.startService();
     }
 
     @Before
     public void setup() throws Exception {
-        createTestEvents(tenantId, 5);
+        createAndInsertTestEvents(tenantId, 5);
         esSetup.client().admin().indices().prepareRefresh().execute().actionGet();
     }
 
     @Test
-    public void TestHttpAnnotationsHappyCase() throws Exception {
+    public void testHttpQueryAnnotationsHappyCase() throws Exception {
+        parameterMap = new HashMap<String, String>();
+        parameterMap.put(Event.fromParameterName, String.valueOf(baseMillis - 86400000));
+        parameterMap.put(Event.untilParameterName, String.valueOf(baseMillis + (86400000*3)));
         HttpGet get = new HttpGet(getAnnotationsQueryURI());
         HttpResponse response = client.execute(get);
+        String responseString = EntityUtils.toString(response.getEntity());
         Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+        Assert.assertFalse(responseString.equals("[]"));
+
+    }
+
+    @Test
+    public void testQueryHttpAnnotationsStaleTimeStamps() throws Exception {
+        parameterMap = new HashMap<String, String>();
+        parameterMap.put(Event.fromParameterName, String.valueOf(baseMillis - 86400000));
+        parameterMap.put(Event.untilParameterName, String.valueOf(baseMillis));
+        HttpGet get = new HttpGet(getAnnotationsQueryURI());
+        HttpResponse response = client.execute(get);
         String responseString = EntityUtils.toString(response.getEntity());
+        Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+        Assert.assertNotNull(responseString);
+        Assert.assertTrue(responseString.equals("[]"));
```

Comment:
No, It's with the stale timestamps so there will be no data in the response payload.